apps/challenges/views.py

```diff
@@ -181,3 +181,16 @@ def get_all_challenges(request, challenge_time):
     except:
         response_data = {'error': 'Wrong url pattern!'}
         return Response(response_data, status=status.HTTP_406_NOT_ACCEPTABLE)
+
+def handler404(request):
+    response = render_to_response('src/views/web/error/404.html', {},
+                                  context_instance=RequestContext(request))
+    response.status_code = 404
+    return response
+
+
+def handler500(request):
+    response = render_to_response('src/views/web/error/500.html', {},
+                                  context_instance=RequestContext(request))
+    response.status_code = 500
+    return response
```

Comment:
@anweshknayak you don't need to write this piece of code in all the repositories. You can just create this in evalai/views.py and put a route for the same in evalai/urls.py